Makefile

```diff
@@ -7,6 +7,8 @@ GO111MODULE=on
 IMAGE=amazon/app-mesh-controller
 REPO=$(AWS_ACCOUNT).dkr.ecr.$(AWS_REGION).amazonaws.com/$(IMAGE)
 VERSION=v0.3.0
+GIT_COMMIT_SHORT=$(shell git log -1 --format=%h)
+DEV_VERSION=${VERSION}-${GIT_COMMIT_SHORT}
```

Comment:
I think its better to use to include information about recent tag and dirty working tree:
```
$ git describe --dirty --tags
v0.0.10-22-gcee18f5-dirty
```

I guess the only potential issue is using VERSION vs the most recent git tag, but I think it should be ok referring to the most recent git tag which should be present due to the github release.

Reply:
TIL, thanks for that cmd.